### PR TITLE
Upgrade astroid and pylint member to reflect current team

### PR DESCRIPTION
**Pull request by Pierre-Sassoulas**
I also took the liberty to put astroid and pylint together.

### CODE REVIEW

File: source/management.rst [hunk 1]
```diff
@@ -20,9 +20,11 @@
...
 - PyLint Administrator(s):
 
-  * Claudiu Popa
-  * Florian Bruhin
-  * Sylvain Thénault
+  * `Claudiu Popa <https://github.com/PCManticore>`_
+  * `Florian Bruhin <https://github.com/The-Compiler>`_
+  * `Sylvain Thénault <https://github.com/sthenault>`_
+  * `Guillaume Peillex <https://github.com/hippo91>`_
+  * `Pierre Sassoulas <https://github.com/Pierre-Sassoulas>`_
```
Comment (sigmavirus24) [inline source/management.rst:27]: Why are these links when nothing else is here?

File: source/introduction.rst [hunk 2]
```diff
@@ -22,30 +22,26 @@ easy to remember namespace for their projects.
...
+    - Developers:
+          * `Marc Mueller <https://github.com/cdce8p>`_
+          * `ceridwen <https://github.com/ceridwen>`_
+          * `Dmitry Pribysh <https://github.com/dmand>`_
+          * `jcristau <https://github.com/jcristau>`_
+          * `Sushobhit <https://github.com/sushobhit27>`_
+          * `Florian Bruhin <https://github.com/The-Compiler>`_
+          * `Łukasz Rogalski <https://github.com/rogalski>`_
+          * `Roy Williams <https://github.com/rowillia>`_
+          * `Sylvain Thénault <https://github.com/sthenault>`_
```
Comment (sigmavirus24) [inline source/introduction.rst:44]: Please follow the style of the rest of the file. (new lines between list elements)

**Review by sigmavirus24 — CHANGES_REQUESTED** [review]

File: source/management.rst (hunk 1 above)
Comment (Pierre-Sassoulas) [reply source/management.rst]: Fixed.

File: source/introduction.rst (hunk 2 above)
Comment (Pierre-Sassoulas) [reply source/introduction.rst]: Fixed

**Pull request merged**